### PR TITLE
tidb-in-kubernetes: add permitHost document

### PR DESCRIPTION
**Pull request by shonge**
### What is changed, added or deleted? 
tidb-in-kubernetes document: add permithost document.

### What is the related PR or file link(s)? 
https://github.com/pingcap/tidb-operator/issues/733

### Which version does your change affect? 
tidb-operator v1.0 +

### CODE REVIEW

File: v3.0/tidb-in-kubernetes/initialize-cluster.md [hunk 1]
```diff
@@ -36,7 +36,17 @@ aliases: ['/docs-cn/v3.0/how-to/configure/initialize-cluster/']
 
     该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
 
-2. 部署集群
+2. 设置允许访问 TiDB 的主机
+
+    在部署集群前可以通过设置 `permitHost` 设置允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
+
+    ```shell
```
Comment (yikeke) [inline v3.0/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
    ```shell
```
`permitHost` 设置是shell里的命令吗

File: v3.0/tidb-in-kubernetes/initialize-cluster.md [hunk 2]
```diff
@@ -36,7 +36,17 @@ aliases: ['/docs-cn/v3.0/how-to/configure/initialize-cluster/']
 
     该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
 
-2. 部署集群
+2. 设置允许访问 TiDB 的主机
+
+    在部署集群前可以通过设置 `permitHost` 设置允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```
Comment (yikeke) [inline v3.0/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
    在部署集群前可以通过设置 `permitHost` 允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```

File: dev/tidb-in-kubernetes/initialize-cluster.md [hunk 3]
```diff
@@ -35,7 +35,17 @@ category: how-to
 
     该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
 
-2. 部署集群
+2. 设置允许访问 TiDB 的主机
+
+    在部署集群前可以通过设置 `permitHost` 设置允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
    在部署集群前可以通过设置 `permitHost` 允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```

Comment (weekface) [inline dev/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
    在部署集群前可以通过设置 `tidb.permitHost` 设置允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
```

File: dev/tidb-in-kubernetes/initialize-cluster.md [hunk 4]
```diff
@@ -35,7 +35,17 @@ category: how-to
...
+
+    ```shell
+    tidb:
+      passwordSecretName: tidb-secret
+      permitHost: <Mysql-client-host-name>
```
Comment (weekface) [inline dev/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
      permitHost: <mysql-client-host-name>
```

File: dev/tidb-in-kubernetes/initialize-cluster.md (hunk 3 above)
Comment (DanielZhangQD) [reply dev/tidb-in-kubernetes/initialize-cluster.md]: Suggest keeping the original description.

**Comment by weekface** [comment]
Please do not merge this pr before: https://github.com/pingcap/tidb-operator/pull/779

File: dev/tidb-in-kubernetes/initialize-cluster.md [hunk 5]
```diff
@@ -33,9 +33,19 @@ category: how-to
     kubectl create secret generic tidb-secret --from-literal=root=<root-password> --from-literal=developer=<developer-passowrd> --namespace=<namespace>
     ```
 
-    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
+    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。并且创建的普通用户`developer`默认只有`USAGE`权限，其他权限请在`tidb.initSql`中设置。
```
Comment (lilin90) [inline dev/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。并且创建的普通用户 `developer` 默认只有 `USAGE` 权限，其他权限请在 `tidb.initSql` 中设置。
```

File: v3.0/tidb-in-kubernetes/initialize-cluster.md [hunk 6]
```diff
@@ -34,9 +34,19 @@ aliases: ['/docs-cn/v3.0/how-to/configure/initialize-cluster/']
     kubectl create secret generic tidb-secret --from-literal=root=<root-password> --from-literal=developer=<developer-passowrd> --namespace=<namespace>
     ```
 
-    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。
+    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。并且创建的普通用户developer默认只有USAGE权限，其他权限请在tidb.initSql中设置。
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/initialize-cluster.md]: - Keep the format consistent.
- Leave a space between Chinese and English characters.

```suggestion
    该命令会创建 `root` 和 `developer` 两个用户的密码，存到 `tidb-secret` 的 Secret 里面。并且创建的普通用户 `developer` 默认只有 `USAGE` 权限，其他权限请在 `tidb.initSql` 中设置。
```

File: v3.0/tidb-in-kubernetes/initialize-cluster.md [hunk 7]
```diff
@@ -44,7 +44,17 @@ aliases: ['/docs-cn/v3.0/how-to/configure/initialize-cluster/']
...
+
+2. 设置允许访问 TiDB 的主机
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
3. 设置允许访问 TiDB 的主机
```

File: v3.0/tidb-in-kubernetes/initialize-cluster.md [hunk 8]
```diff
@@ -44,7 +44,17 @@ aliases: ['/docs-cn/v3.0/how-to/configure/initialize-cluster/']
...
+
+    在部署集群前可以通过 `tidb.permitHost` 配置项来设置允许访问 TiDB 的主机 **host_name**。如果不设置，则允许所有主机访问。详情请参考 [Mysql GRANT host name](https://dev.mysql.com/doc/refman/5.7/en/grant.html)。
+
+    ```
+    tidb:
+      passwordSecretName: tidb-secret
+      permitHost: <Mysql-client-host-name>
+    ```
 
 3. 部署集群
```
Comment (lilin90) [inline v3.0/tidb-in-kubernetes/initialize-cluster.md]:
```suggestion
4. 部署集群
```

**Pull request merged**